src/sql/workbench/contrib/notebook/browser/notebook.contribution.ts

```diff
@@ -193,7 +193,12 @@ configurationRegistry.registerConfiguration({
 			'type': 'boolean',
 			'default': true,
 			'description': localize('notebook.sqlStopOnError', "SQL kernel: stop Notebook execution when error occurs in a cell.")
-		}
+		},
+		'notebook.allowCommandExecution': {
+			'type': 'boolean',
+			'description': localize('notebook.allowCommandExecution', "Allow notebooks to execute Azure Data Studio Commands using the %%AZDATA_EXECUTE_COMMAND magic."),
```

Comment:
Any reason not to just have this bundled up into trust?

Reply:
Notebook trust? I'm not sure that I'm following you. Can you explain? Just adding this as extra customization where users can choose to turn this feature off (e.g. if they want to ensure that their notebooks have compatibility with other viewers, or if they prefer notebooks not be able to run commands in code cells)

Reply:
I'm saying that have it so that if they trust a notebook it'll run the commands. Why the need to split it out of that when we already have this concept of "don't run bad stuff unless you trust the author"?